noxfile.py

```diff
@@ -51,13 +51,12 @@
         "tests",
         "alembic",
     ],
-    "3.10": [
+    "3.11": [
```

Comment:
@tswast 

What are our guidelines on what goes into the Unit Test Extras?
Are those always used solely in conjunction with the earliest and latest Python Versions?

Reply:
Definitely lowest so that we make sure the lower dependency bounds we have in `setup.py` and our contraints files are correct. Latest SGTM so long as the extra dependencies have been released. In the past we'd update before pyarrow had cut a release, but this doesn't appear to be the case right now.